.env-cmdrc

```diff
@@ -1,6 +1,8 @@
 {
   "dev": {
-    "CX_BASE_URL": "https://40.76.109.9:9002"
+    "CX_BASE_URL": "https://localhost:9002",
```

Comment:
I suspect that the changes to this file should be restored, apart from possibly adding a new "S4OM" profile

Reply:
agreed @Pucek9, this was added by accident, will revert the changes